packages/wrangler/src/event-subscriptions/products.ts

```diff
@@ -0,0 +1,74 @@
+import type { HandlerContext } from "../core/types";
+
+// Use the same `id` here as the one you used in queue-broker-worker
+export const PRODUCTS: ProductSpec[] = [
```

Comment:
So the idea is that CF products "onboard" on to the event hub by defining the events they're going to emit. They do so by sending a PR to the Queues codebase. However, there are UI-like things like a resource name that doesn't make sense to put inside Queues. I've got a similar array on the dashboard that basically lets us keep user-facing output here directly.

More importantly though, to create a subscription, Wrangler (and the dashboard) are supposed to send a request to the _product_, and so every product needs to tell Wrangler how to make this request.

This is why I maintain this static `PRODUCTS` list here.

Reply:
can you please add a comment explaining what the purpose of `PRODUCTS` is :pray:. This will help folks outside of your team get some more context <3